frameworks/elastic/tests/test_tls.py

```diff
@@ -79,16 +69,25 @@ def test_crud_over_tls(elastic_service_tls):
     config.create_index(
         config.DEFAULT_INDEX_NAME,
         config.DEFAULT_SETTINGS_MAPPINGS,
-        service_name=PACKAGE_NAME)
+        service_name=SERVICE_NAME,
+        https=True)
     config.create_document(
         config.DEFAULT_INDEX_NAME,
         config.DEFAULT_INDEX_TYPE,
         1,
         {"name": "Loren", "role": "developer"},
-        service_name=PACKAGE_NAME)
-    config.get_document(
+        service_name=SERVICE_NAME,
+        https=True)
+    document = config.get_document(
         config.DEFAULT_INDEX_NAME,
         config.DEFAULT_INDEX_TYPE,
-        1)
+        1,
+        https=True)
+
+    assert document
+    assert document['_source']['name'] == 'Loren'
+
     config.delete_index(
-        config.DEFAULT_INDEX_NAME, service_name=PACKAGE_NAME)
+        config.DEFAULT_INDEX_NAME,
+        service_name=SERVICE_NAME,
+        https=True)
```

Comment:
Is it required to check the success of the delete operation?

Reply:
Not required, no. You can probably remove the call as you've already tested CRUD by this point, and you're just going to tear everything down afterwards. Or move it up before `config.create_index` in case you want to ensure the index doesn't already exist from some prior run.